.github/workflows/build.yml

```diff
@@ -51,4 +51,35 @@ jobs:
           else
             npm i
           fi
-          npm run build
\ No newline at end of file
+          npm run build
+
+  dependencies-license-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Get explicit and transitive dependencies
+      run: |
+        pip install -r requirements.txt
+        pip freeze > requirements-all.txt
+    - name: Check python
+      id: license_check_report
+      uses: pilosus/action-pip-license-checker@v0.6.1
+      with:
+        requirements: 'requirements-all.txt'
+        fail: 'Copyleft,Other,Error'
```

Comment:
Can we change to an include(white)-list approach and not an an exclude(black)-list one?

(or alternatively have the 'fail' categories interpreted dynamically somehow as all of the categories that aren't Permissive. In case they change...)

Reply:
No option, but it includes all non-permissive